Xamarin.Forms.Platform.Android/AppCompat/ImageButtonRenderer.cs

```diff
@@ -83,6 +83,14 @@ protected override void Dispose(bool disposing)
 
 			if (disposing)
 			{
+				if (Element != null)
+				{
+					Element.PropertyChanged -= OnElementPropertyChanged;
```

Comment:
I very much like this pattern of unsubscribing all the event handlers before unhooking other stuff.